### ISSUE TITLE
csi-controller exiting on freenas ZFS and NFS

### ISSUE THREAD

**Issue by diogodias24** [democratic-csi/democratic-csi#207]
Hi,

I noticed an error going on on all my controllers (iscsi and nfs). Container "external provisioner is exiting with:

```console
1 connection.go:187] GRPC error: rpc error: code = Internal desc = ReferenceError: GrpcError is not defined ReferenceError: GrpcError is not defined
    at Api.getSystemVersion (/home/csi/app/src/driver/freenas/http/api.js:284:5)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Api.getIsScale (/home/csi/app/src/driver/freenas/http/api.js:160:27)
    at async FreeNASApiDriver.Probe (/home/csi/app/src/driver/freenas/api.js:2126:13)
    at async requestHandlerProxy (/home/csi/app/bin/democratic-csi:217:18)
```

Using:
* Kubeadm 1.23.7
* Ubuntu 20.04.4 LTS
* Chart Version 0.13.1 (upgraded because had the same behaviour on 0.12)

That block me for providing new volumes using that provisioner. Existing ones still works.

Any other info you will need?

**Comment by travisghansen** [comment]
Looks like there's some issue accessing the api. There is a bug in the code but it won't fix your issue because that code should generally never get hit if things are functioning properly. I'll fix it and release `v1.7.1` with the fix but you probably will want to get the logs from the `csi-driver` container (and possibly enable `debug` logging as well) in order to see what's really going on.

**Comment by travisghansen** [comment]
`v1.7.1` released with a fix for the undefined reference.

**Comment by diogodias24** [comment]
> Looks like there's some issue accessing the api. There is a bug in the code but it won't fix your issue because that code should generally never get hit if things are functioning properly. I'll fix it and release `v1.7.1` with the fix but you probably will want to get the logs from the `csi-driver` container (and possibly enable `debug` logging as well) in order to see what's really going on.

Yep you where right. API Key somehow disappeared from trueness (maybe in the upgrade, or even some rollback I had to do because nfs malfunctioning).

Anyways, created new API Key, applied it on values.yaml and good to go.
You can close this if you want.

Thanks
